### DISCUSSION TITLE
Github Pages and GDPR

### DISCUSSION THREAD

**Discussion by bretg** [community/community#22277] [Other Feature Feedback, Questions, & Ideas]
<p>This topic came up before in <a href="https://github.community/t/how-to-deal-with-gdpr-on-github-pages-based-blogs/10373">https://github.community/t5/GitHub-Pages/How-to-deal-with-GDPR-on-github-pages-based-blogs/m-p/7719</a> but a couple of key points weren’t answered satisfactorily, IMO.</p>
<p>One has to assume that github pages keeps a log of requests for each page that includes user IP address, timestamp, URL, etc.</p>
<p>How long are those logs retained, and how can a site owner request to have entries in those logs removed as per the GDPR requirement for “right to be forgotten”?</p>
<p>thanks</p>

**Comment by bretg** [comment, marked as answer]
<p>Followup – got a response from the Github support team. GDPR support (including Data Protection Agreement) is available for corporate accounts that sign the corporate terms of service. <a href="https://help.github.com/articles/upgrading-to-the-corporate-terms-of-service/" rel="nofollow noopener">https://help.github.com/articles/upgrading-to-the-corporate-terms-of-service/</a></p>

**Comment by loilo** [comment]
<p>Did they also drop anything about private users or developers outside an organization context?</p>
<p>GDPR does apply at least to those devs as well, and this actually sounds like those devs wouldn’t be able to host anything on GitHub Pages for an EU audience.</p>

**Comment by Mansi1** [comment]
Is it possible that we select the location, host country, for the GithubPages. So that our pages are compliant with the GDPR.
